docs/design.adoc

```diff
@@ -290,6 +290,31 @@ being liquidated outside of the coverage pool.
 If both fixed amount and percentage values are set and bigger than zero,
 the fixed amount reward takes precedence.
 
+=== Auction length upgradability
```

Comment:
Can we name this section "Auction length value" so that it is not confused with the upgradeability mechanism between asset pools?

Reply:
Sure, done in 1663b7f